iphone/Maps/Classes/MapViewController.mm

```diff
@@ -374,13 +375,42 @@ - (void)viewDidLayoutSubviews {
   [super viewDidLayoutSubviews];
   if (!self.mapView.drapeEngineCreated && !MapsAppDelegate.isDrapeDisabled)
     [self.mapView createDrapeEngine];
+
+  [self layoutCarplayPlaceholderButton];
+}
+
+- (void)layoutCarplayPlaceholderButton {
+  CGFloat buttonWidth = self.view.frame.size.width - 16 * 2;
+  CGFloat buttonHeight = [self.carplayPlaceholderButton sizeThatFits:CGSizeMake(buttonWidth, 0)].height;
+  self.carplayPlaceholderButton.frame = CGRectMake(
```

Comment:
I don't like storyboards a lot, but it may be better to add this button to the storyboard right below the logo with constraints. Because this controller isn't layouted programmatically (except the map and place page which cover the entire screen).

Reply:
>Would it be better to make this button more visible?

Agree, I think it would be better to match with android UI, in that case we could also reuse all strings. I will look at this.

Reply:
Android UI for reference:

<img width="360" alt="android" src="https://github.com/organicmaps/organicmaps/assets/1800899/0a08e2d9-cf7c-4465-ac9a-373bee41098d">
<img width="552" alt="android_horiz" src="https://github.com/organicmaps/organicmaps/assets/1800899/ce88d119-5b35-4cd2-920c-76eb66338429">

Reply:
Here is result

Regular/pressed state:
<img src="https://github.com/organicmaps/organicmaps/assets/1800899/43f71ba4-77f3-4a3a-8aa0-6e433c5bc824" width="200"> <img src="https://github.com/organicmaps/organicmaps/assets/1800899/196f2443-0480-4531-87ce-ca9dd7730a79" width="200">


Horizontal:
<img src="https://github.com/organicmaps/organicmaps/assets/1800899/eadf1b59-3973-4ceb-8525-eb6cc6ba1595" width="600">